cpp/src/preprocessor.cpp

```diff
@@ -44,7 +44,7 @@ void PreProcessor::process(std::vector<std::string> lines) {
 
   int new_indent;
 
-  for (int line_number = 0; line_number < lines.size(); line_number++) {
+  for (unsigned int line_number = 0; line_number < lines.size(); line_number++) {
```

Comment:
lol no idea, must have been debugging something. no warnings or anything without it. gone now